templates/list.jsx

```diff
@@ -53,17 +53,13 @@ export default function List({ _columns, _orderedList, _items, ...props }) {
                       role="heading"
                       aria-level={a11y.ariaLevel({ id: _id, level: 'componentItem', override: _ariaLevel ?? itemAriaLevel })}
                     >
-                      <div className="list-item__title-inner">
-                        {html(compile(title))}
-                      </div>
+                      <div className="list-item__title-inner" dangerouslySetInnerHTML={{ __html: title }} />
                     </div>
                   }
 
                   {body &&
                     <div className="list-item__body">
-                      <div className="list-item__body-inner">
-                        {html(compile(body))}
-                      </div>
+                      <div className="list-item__body-inner" dangerouslySetInnerHTML={{ __html: body }} />
```

Comment:
Perhaps amend to `compile(body)` so that Handlebars expressions are still dealt with